crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -492,42 +492,49 @@ where
     fn emit_str_lit(&mut self, node: &Str) -> Result {
         self.emit_leading_comments_of_span(node.span(), false)?;
 
-        let (single_quote, value) = match node.kind {
+        match node.kind {
             StrKind::Normal { contains_quote } => {
-                let single_quote = if contains_quote {
-                    is_single_quote(&self.cm, node.span)
-                } else {
-                    None
-                };
+                let is_invalid_use_strict = node.has_escape && &*node.value == "use strict";
```

Comment:
It will be improved when we will have `raw`, because we will know how developer writes it, currently we need this hack